### ISSUE TITLE
Notification not sent when updating issue

### ISSUE THREAD

**Issue by gyscos** [sciyoshi/redmine-slack#123]
Environment:
```
Environment:
  Redmine version                3.4.2.stable
  Ruby version                   2.4.1-p111 (2017-03-22) [x86_64-linux]
  Rails version                  4.2.8
  Environment                    production
  Database adapter               SQLite
SCM:
  Subversion                     1.8.10
  Mercurial                      3.1.2
  Bazaar                         2.7.0
  Git                            2.1.4
  Filesystem                     
Redmine plugins:
  redmine_contacts               4.1.1
  redmine_slack                  0.2
```

Notifications are correctly sent when creating new issues, but not when updating issues (for instance when adding a comment to an issue).

The configuration page for the slack plugin shows both `Post Issue Updates?` and `Post Wiki Updates?` as checked.

Note: wiki updates also do not trigger notifications.

**Comment by gyscos** [comment]
EDIT: looks like #107, and should be fixed by #121.

**Comment by sciyoshi** [comment]
Fixed by #121 - please update for the fix.
